src/main/java/Duke.java

```diff
@@ -1,10 +1,244 @@
+<<<<<<< HEAD
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.ChronoUnit;
+=======
+import java.io.File;
+import java.io.IOException;
+>>>>>>> branch-Level-7
+
+import static java.lang.Integer.parseInt;
+
 public class Duke {
+    //@@author IceFire
+    //Reused from https://stackoverflow.com/questions/36514289
+    // with minor modifications
+    private static String dashedLine()
+    {
+        StringBuilder sb = new StringBuilder(90);
+        for (int i = 0; i < 90; i++) {
+            sb.append("-");
+        }
+        return sb.toString();
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
+        /*String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+                + "|____/ \\__,_|_|\\_\\___|\n";*/
+
+        Scanner userInput = new Scanner(System.in);
+        System.out.println(dashedLine());
+        System.out.println("My name is Skyler White yo \n"
+                + "How can I help you?");
+        System.out.println(dashedLine());
+        ArrayList<Task> list = new ArrayList<>();
+        boolean end = false;
+        while(!end) {
+            System.out.print("Please Input: ");
+            end = processInput(userInput.nextLine(), list, end);
+        }
+    }
+
+    private static boolean processInput(String input, ArrayList<Task> list, boolean end) {
+        String[] inputAnalyzed = input.split(" ");
+        System.out.println(dashedLine());
+        switch (inputAnalyzed[0].toLowerCase(Locale.ROOT)) {
+            case "bye":
+                end = byeOperation(inputAnalyzed, list);
+                break;
+            case "list":
+                listOperation(inputAnalyzed,list);
+                break;
+            case "mark":
+                markOperation(inputAnalyzed,list);
+                break;
+            case "unmark":
+                unmarkOperation(inputAnalyzed,list);
+                break;
+            case "deadline":
+                ddlOperation(inputAnalyzed,input,list);
+                break;
+            case "todo":
+                todoOperation(inputAnalyzed,input,list);
+                break;
+            case "event":
+                eventOperation(inputAnalyzed,input,list);
+                break;
+            case "delete":
+                deleteOperation(inputAnalyzed,list);
+                break;
+            default:
+                System.out.println("Sorry sir, didn't quite get that." +
+                        "\nYou have " + list.size() + " tasks. Anything else?");
+        }
+        System.out.println(dashedLine());
+        return end;
+    }
+
+    private static boolean byeOperation(String[] inputAnalyzed, ArrayList<Task> list){
```

Comment:
In terms of code quality/readability, it may be better to use verbs for method names.